src/ListClient.ts

```diff
@@ -4,7 +4,9 @@ import invariant from 'tiny-invariant';
 import { LocalBus } from './localbus';
 import { ListInterpreter, ListMeta, ListStore } from '@roomservice/core';
 
-export class InnerListClient<T extends any> implements ObjectClient {
```

Comment:
@kcking if you do this, it both won't match the map client, where you pass in the whole object instead of the property value.

Reply:
oh i see, what if we passed `T[number]` to all of the `ListInterpreter` methods?

Reply:
(and reverted back to `T extends Array<any>`)

Reply:
> oh i see, what if we passed T[number] to all of the ListInterpreter methods?

That should work :+1: